.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -41,7 +41,15 @@ You can get the script and run it with:
 ```bash
 wget https://raw.githubusercontent.com/Lightning-AI/lightning/master/requirements/collect_env_details.py
 python collect_env_details.py
+
 ```
+And paste it below inside the Details summary toggle.
+
+<details>
+  <summary>Details</summary>
+
```

Comment:
Maybe we should put the indicator text exactly here where they should paste the content?

Reply:
@awaelchli Agree!
But at the same time, keeping the toggle inside the comment block might be not very intuitive...
A user can paste the results inside the toggle and forget to move the toggle outside the comment block.